addons/pos_loyalty/static/src/overrides/components/control_buttons/control_buttons.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates id="template" xml:space="preserve">
     <t t-name="pos_loyalty.ControlButtons" t-inherit="point_of_sale.ControlButtons" t-inherit-mode="extension">
-        <xpath expr="//OrderlineNoteButton" position="after">
+        <xpath expr="//t[@t-if='props.wrapped || (ui.isSmall and !props.wrapped)']/OrderlineNoteButton" position="after">
```

Comment:
```suggestion
        <xpath expr="//OrderlineNoteButton" position="after">
```

I don't think there is a need to change this line.

Reply:
@caburj 
this one is used to be more accurate in adding the button 
because we have used this condition also to add buttons in dialog pop 
https://github.com/odoo/odoo/pull/167301/files#diff-88f75f5941594ac91e450b290efeb91716a29da1ada446b723bda1a8550d3b76R30